website/docs/guides/migration/versions/upgrading-to-dbt-utils-v1.0.md

```diff
@@ -0,0 +1,116 @@
+# dbt utils 1.0 Migration Guide
+
+For the first time, dbt utils is crossing the major version boundary. From [last month’s blog post](https://www.getdbt.com/blog/announcing-dbt-v1.3-and-utils/): 
+
+> It’s time to formalise what was already unofficial policy: you can rely on dbt utils in the same way as you do dbt Core, with stable interfaces and consistent and intuitive naming.
+
+Just like the switch to dbt Core 1.0 last year, there are some breaking changes as we standardised and prepared for the future. Most changes can be handled with find-and-replace; if you need help, post on the [Community Forum](https://discourse.getdbt.com) or in #package-ecosystem on Slack. 
+
+## New features
+
+- `get_single_value()` is an easy way to pull a single value from a SQL query, instead of having to access the `[0][0]`th element of a `run_query` result.
```

Comment:
```suggestion
- `get_single_value()` &mdash; An easy way to pull a single value from a SQL query, instead of having to access the `[0][0]`th element of a `run_query` result.
```